orchestra/proc-macro/src/impl_orchestra.rs

```diff
@@ -58,10 +57,6 @@ pub(crate) fn impl_orchestra_struct(info: &OrchestraInfo) -> proc_macro2::TokenS
 		// without `cargo fmt`, there will be some weirdness around else brackets
 		// that does not originate from how we create it
 
-		/// Capacity of a bounded message channel between orchestra and subsystem
-		/// but also for bounded channels between two subsystems if not overriden.
-		const CHANNEL_CAPACITY: usize = #message_channel_capacity;
-
 		/// Capacity of a signal channel between a subsystem and the orchestra.
 		const SIGNAL_CHANNEL_CAPACITY: usize = #signal_channel_capacity;
```

Comment:
Is the `const SIGNAL_CHANNEL_CAPACITY` still required?

Reply:
Yes, it is used in the connector from what I see.

Reply:
But shouldn't it be symmetrical at this point with the `CHANNEL_CAPACITY/message_channel_capacity`.

Reply:
Hum, I really don't know tbh. It is defined [here](https://github.com/paritytech/orchestra/blob/master/orchestra/proc-macro/src/impl_builder.rs#L432) and also [here](https://github.com/paritytech/orchestra/blob/master/orchestra/proc-macro/src/impl_builder.rs#L559).

Reply:
So originally I used the constants as intermediary to be able to reason better about the code and be able to see where the values come from in the generated code.

Reply:
Connector can be specified as an external argument, so it cannot really reason about the content of the builder/orchestra structures.

Reply:
The remaining uses of the signal channel capacity are a hack, because we need to construct the the sender before we start creating the builder. And we need that since we need to pass the sender around to some substrate based _thing_. I'd have to check polkadot again to see if the connector / builder split is actually used.

We could create a custom sender like API and internally mutate the thing with a buffer. It's not pretty and quite a bit of code for a questionable gain and also debatable if that's not a misc addition.

Reply:
Let's defer the connector part and focus on getting this PR merged